Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -132,21 +157,59 @@ export type NotificationProps = {
   onHide?: (id: string) => void;
 };
 
+const makeLinkLabel = (link: NotificationLink) => (
```

Comment:
Can you deconstruct link?
```suggestion
const makeLinkLabel = ({ label, onClick }: NotificationLink) => (
```